### ISSUE TITLE
Source-Code encoding can only be specified once

### ISSUE THREAD

**Issue by nedbat** [coveragepy/coveragepy#453]
*Originally reported by* **Anonymous**

----------------------------------------

Hi

I found that coveralls can't process files which have both the emacs and vim style fileencoding specifiers are given at the top of a file.

```
# -*-  coding:utf-8 -*-
# vim: fileencoding=utf-8
```

 I created a [gist](https://gist.github.com/kain88-de/c28935bb00e836b248fd) that reproduces the problem. You can run it with

```
nosetests test_example.py --with-coverage --cover-package=exampl
```

This will result in an error like this.

```
example.py   NotPython: Couldn't parse '/home/max/Desktop/coverage-test/example.py' as Python source: 'encoding declaration in Unicode string' at line 0
```

According to [PEP 0263](https://www.python.org/dev/peps/pep-0263/) both are OK but it
doesn't say anything about giving both forms. 

----------------------------------------
- Bitbucket: https://bitbucket.org/ned/coveragepy/issue/453

**Comment by nedbat** [comment]
Finish up Max Linke's fix for double encoding declarations. Fixes #453

→ <<cset [3d74bc2d93c1 (bb)](https://bitbucket.org/ned/coveragepy/commits/3d74bc2d93c1)>>

**Comment by nedbat** [comment]
Fixed in [3d74bc2d93c1 (bb)](https://bitbucket.org/ned/coveragepy/commits/3d74bc2d93c1)
